spec/models/corporate_information_page_spec.rb

```diff
@@ -30,4 +30,43 @@
       expect(described_class.new(content_store_response).default_organisation).to be_nil
     end
   end
+
+  describe "#corporate_information?" do
+    it "presents corporate information groups on about pages" do
+      expect(described_class.new(content_store_response).corporate_information?).to be(true)
+    end
+
+    it "does not present corporate information groups on about pages if information isn't available" do
+      content_store_response["details"]["corporate_information_groups"] = nil
+
+      expect(described_class.new(content_store_response).corporate_information?).to be(false)
+    end
+  end
+
+  describe "#corporate_information" do
+    it "includes group links that are guids" do
+      presented_groups = described_class.new(content_store_response).corporate_information
```

Comment:
Perhaps set `subject` at the beginning of the describe block so we don't get presenters and models confused in the tests. I had to double check what type of test I was looking at!

e.g.

```ruby
subject(:corporate_information) { described_class.new(content_store_response).corporate_information }
```

then the expectation can use `expect(corporate_information.groups.first` etc